packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.tsx

```diff
@@ -68,7 +70,7 @@ const MobileFullPageModal = ({
                 })}
                 height_offset={height_offset}
             >
-                {(renderPageHeader || page_header_text || renderPageHeaderText) && (
+                {(renderPageHeader || page_header_text || renderPageHeaderText || renderPageHeaderElement) && (
```

Comment:
`renderPageHeaderText` only be used in one place and it also return element there somehow, can we maybe remove it and keep renderPageHeaderElement instead. And change the logic here in line 88
![image](https://github.com/farrah-deriv/deriv-app/assets/125247833/b92b3401-025e-4341-b4e4-7dab87a6b761)

Reply:
sure let me check that